src/hoi4_world/countries/hoi4_country_converter_tests.cpp

```diff
@@ -1114,5 +1114,50 @@ TEST(Hoi4worldCountriesCountryConverter, GraphicsBlocksAreSet)
    EXPECT_THAT(country_one->GetGraphicsBlock().portrait_paths.army, testing::ElementsAre("GFX_general"));
 }
 
+TEST(Hoi4worldCountriesCountryConverter, Puppets)
```

Comment:
Test names should be a sentence fragment that indicates the desired behavior. In this case, something like PuppetsAreConverted